tests/userCreate.test.js

```diff
@@ -0,0 +1,104 @@
+const { createUser } = require("../controllers/userControllers");
+const database = require("../models/User");
+
+jest.mock("../models/User");
+
+describe("user controller", () => {
+  let req, res;
+
+  beforeEach(() => {
+    req = {
+      body: {},
+      param: {},
+    };
+    res = {
+      status: jest.fn().mockReturnThis(),
+      json: jest.fn().mockReturnThis(),
+    };
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
I think we run into an issue for using clear instead of reset.

Reply:
What approach would you use?